conf/pacific/cephadm/tier-1_3node_cephadm_bootstrap.yaml

```diff
@@ -41,3 +41,4 @@ globals:
       node4:
         role:
           - client
+          - rgw
```

Comment:
hope there would be changes here, let's have separate client node.

Reply:
Changes done. Could you please review?